noir-projects/aztec-nr/aztec/src/encrypted_logs/log_type.nr

```diff
@@ -1,3 +1,3 @@
-pub global PRIVATE_NOTE_LOG_TYPE_ID: u64 = 0;
```

Comment:
This file should be renamed (not sure if you had a plan to do it in a followup PR)

Reply:
I was planning on doing it later but yea i'll just stop being lazy.